docs/en/modules/ws2812.md

```diff
@@ -181,6 +181,87 @@ The number of given bytes must match the number of bytesPerLed of the buffer
 buffer:fill(0, 0, 0) -- fill the buffer with black for a RGB strip
 ```
 
+## ws2812.buffer:dump()
+Returns the contents of the buffer (the pixel values) as a string. This can then be saved to a file or sent over a network.
+
+#### Syntax
+`buffer:dump()`
+
+#### Returns
+A string containing the pixel values. 
+
+#### Example
+```lua
+local s = buffer:dump() 
+```
+
+## ws2812.buffer:load()
+Converts a string into a buffer (see also the `dump` method). The buffer must have the correct shape or an error will be thrown.
```

Comment:
I've changed this to be more general purpose and allow an insert operation into the middle of another buffer. Also, the source can either be a buffer or a string.